### PR TITLE
Updated example, added php.ini initialisation

### PR DESCRIPTION
**Pull request by Cawllec**
- Updated example readme and files.
- Adds example of prepending bugsnag initialisation using php.ini to catch all errors & exceptions.

### CODE REVIEW

**Comment by Cawllec** [comment]
Ah, thanks Graham

**Comment by GrahamCampbell** [comment]
> auto_prepend_file=prepend.ph

I wouldn't recommend deploying bugsnag to php apps in that way.

**Comment by GrahamCampbell** [comment]
I'm kinda surprised that ini setting still exists. I think it's likely it will be removed in later php versions.

**Comment by Cawllec** [comment]
Yeah, it's definitely not a recommended step, but several people have asked about using it, so the example is there as a demonstration.  Maybe the language in the readme/docs needs to changed to be more of a recommendation against it.

File: README.md [hunk 1]
```diff
@@ -16,22 +15,29 @@ The Bugsnag error reporter for PHP gives you instant notification of errors and
...
 
 ## Support
 
 * Read the configuration reference:
-    * [Laravel](http://docs.bugsnag.com/platforms/php/laravel/configuration-options)
-    * [Lumen](http://docs.bugsnag.com/platforms/php/lumen/configuration-options)
-    * [Other PHP apps](http://docs.bugsnag.com/platforms/php/other/configuration-options)
+  * [Laravel](http://docs.bugsnag.com/platforms/php/laravel/configuration-options)
+  * [Lumen](http://docs.bugsnag.com/platforms/php/lumen/configuration-options)
+  * [Other PHP apps](http://docs.bugsnag.com/platforms/php/other/configuration-options)
```
Comment (fractalwrench) [inline README.md]: http -> https

File: example/php/catchall.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+syntaxfailure()[]
```
Comment (fractalwrench) [inline example/php/catchall.php]: missing newline

File: example/php/php.ini [hunk 3]
```diff
@@ -0,0 +1 @@
+auto_prepend_file=prepend.php
```
Comment (fractalwrench) [inline example/php/php.ini]: as above

**Comment by Cawllec** [comment]
Could I get a quick review on this @kattrali so I can merge it down?

**Pull request merged**